envsec/internal/auth/user.go

```diff
@@ -73,7 +74,7 @@ func (u *User) OrgID() string {
 	return u.IDToken.Claims.(jwt.MapClaims)["org_id"].(string)
```

Comment:
Are we just reading the `org_id` claim or are we attempting to read `https://auth.jetpack.io/org_id` from the token as well?
If that's the case or if we want to read both, this needs to be updated to accommodate for it. 
Also, can we change all `org_id`'s to `org-id` to align with current implementation of envsec?

Reply:
This function is most likely going away (it's not needed). The org_id is only used by the server (the server should validate the JWT and extract it on it's own, that's the secure way to do it). I'll remove this after I make the server changes